user_guide_src/source/installation/upgrade_emails.rst

```diff
@@ -0,0 +1,60 @@
+Upgrade Emails
+##############
+
+.. contents::
+    :local:
+    :depth: 1
+
+
+Documentations
+==============
+
+- `Email Documentation Codeigniter 3.X <http://codeigniter.com/userguide3/libraries/email.html>`_
+- :doc:`Email Documentation Codeigniter 4.X </libraries/email>`
+
+
+What has been changed
+=====================
+- Only small things like the method names and the loading of the library have changed.
+
+Upgrade Guide
+=============
+1. Within your class change the ``$this->load->library('email');`` to ``$email = \Config\Services::email();``.
```

Comment:
For the rest of these, I would encourage the use of the `service('email')` helper in place of the fully qualified method name. While both are appropriate, using the helper protects for the future in case any small tweaks are made to the Services class in v5 that might break it.

Reply:
```suggestion
1. Within your class change the ``$this->load->library('email');`` to ``$email = services('email');``.
```